fe/projects/AspNetCore/src/shared/components/LanguageSwitcher/LanguageSwitcher.js

```diff
@@ -1,24 +1,53 @@
-import React from "react";
+import React, { useState } from "react";
```

Comment:
**suggestion:** shouldn't the extension be jsx instead of js? Please review everywhere